File: NET_4.5.2/C1.WPF/VB/C1TreeViewDragDropSample/C1TreeViewDragDropSample2010/Department.vb
Imports System.Collections.ObjectModel

Namespace C1TreeViewDragDropSample2010
	''' <summary>
	''' Business object used to represent a Department.
	''' </summary>
	Public Class Department
		Public Sub New()
			Employees = New ObservableCollection(Of Employee)()
		End Sub

		Public Property DepartmentID() As Integer
		Public Property Name() As String
		Public Property GroupName() As String
		Public Property Employees() As ObservableCollection(Of Employee)
	End Class
End Namespace

File: NET_4.5.2/C1.WPF/CS/C1TreeViewDragDropSample/C1TreeViewDragDropSample2010/Department.cs
using System.Collections.ObjectModel;

namespace C1TreeViewDragDropSample2010
{
    /// <summary>
    /// Business object used to represent a Department.
    /// </summary>
    public class Department
    {
        public Department()
        {
            Employees = new ObservableCollection<Employee>();
        }

        public int DepartmentID { get; set; }
        public string Name { get; set; }
        public string GroupName { get; set; }
        public ObservableCollection<Employee> Employees { get; set; }
    }
}
